engine/src/main/java/org/camunda/bpm/engine/impl/interceptor/CommandInvocationContextFactory.java

```diff
@@ -0,0 +1,12 @@
+package org.camunda.bpm.engine.impl.interceptor;
```

Comment:
Please format this file so that it uses two spaces as indentation instead of four.